_includes/footer.html

```diff
@@ -14,5 +14,7 @@
 
   <script src="/js/vendor/jquery-1.10.2.min.js"></script>
   <script src="/js/vendor/bootstrap.min.js"></script>
+  <script src="/js/vendor/anchor.min.js"></script>
+  <script src="/js/app/main.js"></script>
```

Comment:
Could you inline main.js here? The size of the header and overhead of the tcp-ip handshake is probably significantly larger in this case then the size of the file itself:-)